### PR TITLE
docs(README): fix Markdown

### PR DESCRIPTION
**Pull request by MicaelJarniac**
Fixed many broken links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ after_success:
 ```
 
 ### Produce Coverage Reports
-`coverage.py <https://bitbucket.org/ned/coveragepy>`_ is required to collect coverage metrics.
+[`coverage.py`](https://bitbucket.org/ned/coveragepy) is required to collect coverage metrics.
```
[the remark this answers is not in the document]
Comment (thomasrockhu) [reply README.md:18]: Also the link should be `https://github.com/nedbat/coveragepy`

File: README.md [hunk 2]
```diff
@@ -15,13 +15,13 @@ after_success:
...
 
 Below are some examples on how to include coverage tracking during your tests. Codecov will call `coveragexml -i` automatically to generate the coverage xml output, which will be archived and processed server side.
 
-You may need to configure a ``.coveragerc`` file. Learn more `here <http://coverage.readthedocs.org/en/latest/config.html>`_. Start with this `generic .coveragerc <https://gist.github.com/codecov-io/bf15bde2c7db1a011b6e>`_ for example.
+You may need to configure a `.coveragerc` file. Learn more [here](http://coverage.readthedocs.org/en/latest/config.html>). Start with this [generic .coveragerc](https://gist.github.com/codecov-io/bf15bde2c7db1a011b6e>) for example.
```
Comment (thomasrockhu) [inline README.md:22]: no need for `>` in the URL

File: README.md [hunk 3]
```diff
@@ -40,11 +40,11 @@ pytest --cov=./
 ```
 nosetest --with-coverage
 ```
-See the `Offical Nose coverage docs <http://nose.readthedocs.org/en/latest/plugins/cover.html>`_ for more information.
+See the [Offical Nose coverage docs](http://nose.readthedocs.org/en/latest/plugins/cover.html>) for more information.
```
Comment (thomasrockhu) [inline README.md:43]: No need for the trailing `>`

File: README.md [hunk 4]
```diff
@@ -53,12 +53,15 @@ commands = codecov
 ```
 
 ### FAQ
-- Q:  What's the difference between the codecov-bash and codecov-python uploader?<br/>A: As far as python is concerned, *nothing*. You may choose to use either uploader. Codecov recommends **using the bash uploader when possible** as it supports more unique repository setups. Learn more at `codecov/codecov-bash <https://github.com/codecov/codecov-bash>`_ and `codecov/codecov-python <https://github.com/codecov/codecov-python>`_.
-- Q:  Why am I seeing `No data to report`?<br/>A: This output is written by running the command ``coverage xml`` and states that there were no ``.coverage`` files found.
+- Q:  What's the difference between the codecov-bash and codecov-python uploader?  
+  A: As far as python is concerned, *nothing*. You may choose to use either uploader. Codecov recommends **using the bash uploader when possible** as it supports more unique repository setups. Learn more at [codecov/codecov-bash](https://github.com/codecov/codecov-bash>) and [codecov/codecov-python](https://github.com/codecov/codecov-python>).
```
Comment (thomasrockhu) [inline README.md:57]: Trailing `>`s

File: README.md [hunk 5]
```diff
@@ -53,12 +53,15 @@ commands = codecov
...
+- Q:  Why am I seeing `No data to report`?  
+  A: This output is written by running the command `coverage xml` and states that there were no `.coverage` files found.
 	1. Make sure coverage is enabled. See Enabling Coverage
 	2. You may need to run `coverage combine` before running Codecov.
-	3. Using Docker? Please follow this step: `Testing with Docker: Codecov Inside Docker <https://github.com/codecov/support/wiki/Testing-with-Docker#codecov-inside-docker>`_.
-- Q: Can I upload my ``.coverage`` files?<br/> A: **No**, these files contain coverage data but are not properly mapped back to the source code. We rely on ``coveragepy`` to handle this by calling ``coverage xml`` in the uploader.
+	3. Using Docker? Please follow this step: [Testing with Docker: Codecov Inside Docker](https://github.com/codecov/support/wiki/Testing-with-Docker#codecov-inside-docker>).
```
Comment (thomasrockhu) [inline README.md:62]: Trailing `>`

**Review by thomasrockhu — CHANGES_REQUESTED** [review]
Thanks for the PR @MicaelJarniac. Looks like there are still some outstanding URLs that need to be fixed.

File: README.md [hunk 6]
```diff
@@ -53,12 +53,15 @@ commands = codecov
...
+- Q:  Why am I seeing `No data to report`?  
+  A: This output is written by running the command `coverage xml` and states that there were no `.coverage` files found.
 	1. Make sure coverage is enabled. See Enabling Coverage
 	2. You may need to run `coverage combine` before running Codecov.
-	3. Using Docker? Please follow this step: `Testing with Docker: Codecov Inside Docker <https://github.com/codecov/support/wiki/Testing-with-Docker#codecov-inside-docker>`_.
-- Q: Can I upload my ``.coverage`` files?<br/> A: **No**, these files contain coverage data but are not properly mapped back to the source code. We rely on ``coveragepy`` to handle this by calling ``coverage xml`` in the uploader.
+	3. Using Docker? Please follow this step: [Testing with Docker: Codecov Inside Docker](https://github.com/codecov/support/wiki/Testing-with-Docker#codecov-inside-docker).
```
Comment (MicaelJarniac) [inline README.md:62]: This URL seems dead, I have no idea what to do with it...
https://github.com/codecov/support/wiki/Testing-with-Docker#codecov-inside-docker

**Review by MicaelJarniac — COMMENTED** [review]
I've removed the trailing `>`s and changed the coverage.py URL to point to the correct GitHub repo.

File: README.md (hunk 6 above)
Comment (thomasrockhu) [reply README.md:62]: I think this should be https://docs.codecov.io/docs/testing-with-docker#codecov-inside-docker

Comment (MicaelJarniac) [reply README.md]: Fixed it!

**Review by thomasrockhu — APPROVED** [review]
Thanks @MicaelJarniac!

**Pull request merged**